operator/pkg/apis/istio/v1alpha1/values_types.proto

```diff
@@ -723,6 +723,8 @@ message PilotConfig {
   //
   // See: https://kubernetes.io/docs/tutorials/security/seccomp/
   google.protobuf.Struct seccompProfile = 38;
+
+  google.protobuf.Struct affinity = 40;
```

Comment:
I think this can be set in `pilot.k8s.affinity`, have you tried this?

Reply:
> I think this can be set in `pilot.k8s.affinity` in operator spec, have you tried this?

it does not introduce the k8s structure in this file , just like

google.protobuf.Struct tolerations   =>  k8s.io.api.core.v1.Toleration tolerations

google.protobuf.Struct volumes  =>  k8s.io.api.core.v1.Volume volumes

so  it's better to be consistent

Reply:
just like `tolerations` ` volumes` 

it's better to be consistent